<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTKMainStackForm</class>
 <widget class="QWidget" name="PTKMainStackForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <item>
      <property name="text">
       <string>Основная страница</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ввод данных</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Группирование</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Обработка</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="mainStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="MainVLayout">
         <item>
          <widget class="QPushButton" name="inputLoadTransportBtn">
           <property name="text">
            <string>Ввести входные данные для погрузочных транспортных средств</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inputTipperBtn">
           <property name="text">
            <string>Ввести входные данные для самосвалов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="processingBtn">
           <property name="text">
            <string>Обработать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inputDataPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Вводные данные</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="importExportHLayout">
         <item>
          <widget class="QPushButton" name="importDataButton">
           <property name="text">
            <string>Импортировать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportDataButton">
           <property name="text">
            <string>Эскпортировать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="inputCellsVLayout">
         <item>
          <layout class="QVBoxLayout" name="firstInputVLayout"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="secondInputVLayout"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="nextPageHLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="backToMainPage">
           <property name="text">
            <string>Вернуться на основную страницу</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="processingPage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="processingLbl">
           <property name="text">
            <string>Обработка</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="loadTransportLbl">
           <property name="text">
            <string>Количество транспортных средств для загрузки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tipperCountLbl">
           <property name="text">
            <string>Количество автосамосвалов:</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="groupDataButton">
           <property name="text">
            <string>Сгруппировать ПТК</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="processingBtn_2">
           <property name="text">
            <string>Обработать</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="groupDataPage">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Группировка</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QListView" name="tippersListView"/>
           </item>
           <item>
            <widget class="QListView" name="loatTransportListView"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="createPTKBtn">
           <property name="text">
            <string>Создать ПТК</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backToProcessingPage">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
